client/signup/steps/site/index.jsx

```diff
@@ -1,26 +1,26 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	isEmpty = require( 'lodash/isEmpty' ),
-	includes = require( 'lodash/includes' ),
-	map = require( 'lodash/map' ),
-	debug = require( 'debug' )( 'calypso:steps:site' ); // eslint-disable-line no-unused-vars
+import React from 'react';
+import isEmpty from 'lodash/isEmpty';
```

Comment:
And here, in this case we would see more benefit as we could truncate the 3 imports :D
Actually, this might be a good thing to tackle on another pass, regexing or code-modding for `lodash/xx` specifically.